lib/archethic/contracts/interpreter/common_interpreter.ex

```diff
@@ -272,9 +275,12 @@ defmodule Archethic.Contracts.Interpreter.CommonInterpreter do
   #  |_|
   # ----------------------------------------------------------------------
   # exit block == set parent scope
+  def postwalk(_, _, function_keys \\ [])
```

Comment:
I think this default value is not necessary since the function_key are always passed as parameter